
package kz.tamur.shep.asynchronous;

import javax.xml.ws.WebFault;


/**
 * Ошибка
 *
 * This class was generated by Apache CXF 2.4.6
 * 2014-09-27T20:09:31.039+06:00
 * Generated source version: 2.4.6
 */

@WebFault(name = "sendMessageFault1_sendMessageFault", targetNamespace = "http://bip.bee.kz/AsyncChannel/v10/Types")
public class SendMessageSendMessageFaultMsg extends Exception {
    
    private kz.tamur.shep.asynchronous.ErrorInfo sendMessageFault1SendMessageFault;

    public SendMessageSendMessageFaultMsg() {
        super();
    }
    
    public SendMessageSendMessageFaultMsg(String message) {
        super(message);
    }
    
    public SendMessageSendMessageFaultMsg(String message, Throwable cause) {
        super(message, cause);
    }

    public SendMessageSendMessageFaultMsg(String message, kz.tamur.shep.asynchronous.ErrorInfo sendMessageFault1SendMessageFault) {
        super(message);
        this.sendMessageFault1SendMessageFault = sendMessageFault1SendMessageFault;
    }

    public SendMessageSendMessageFaultMsg(String message, kz.tamur.shep.asynchronous.ErrorInfo sendMessageFault1SendMessageFault, Throwable cause) {
        super(message, cause);
        this.sendMessageFault1SendMessageFault = sendMessageFault1SendMessageFault;
    }

    public kz.tamur.shep.asynchronous.ErrorInfo getFaultInfo() {
        return this.sendMessageFault1SendMessageFault;
    }
}
